templates/details/resources.html

```diff
@@ -114,23 +118,64 @@ <h3 class="p-heading--5">Release history</h3>
     panelToggleButtons.forEach(function(panelToggleButton) {
       panelToggleButton.addEventListener("click", function(e) {
         var target = e.target;
-        var panel = panelToggleButton.querySelector(".p-inline-list__panel");
+        var panel = document.getElementById(
+          target.getAttribute("aria-controls")
+        );
 
-        if (target !== panel) {
-          panelToggleButton.classList.toggle("is-open");
+        target.classList.toggle("is-active");
+        panel.classList.toggle("is-open");
+
+        if (panel.classList.contains("is-open")) {
+          panel.querySelector("[readonly]").focus();
         }
       });
     });
 
-    document.addEventListener("click", function(e) {
-      var target = e.target;
-      var openPanelToggle = document.querySelector(".js-panel-toggle.is-open");
 
-      if (openPanelToggle) {
-        var openPanel = openPanelToggle.querySelector(".p-inline-list__panel");
+    document.addEventListener("keydown", function(e) {
+      var escKeyCode = 27;
+      var tabKeyCode = 9;
+
+      var openPanelToggle = document.querySelector(
+        ".js-panel-toggle.is-active"
+      );
+
+      var openPanel = document.querySelector("[readonly]");
+
+      if (e.keyCode === escKeyCode) {
```

Comment:
Can you not just check both here? `e.keyCode === escKeyCode || e.keyCode === tabKeyCode` that would remove the duplication in the code.

Reply:
@anthonydillon yes, I'm in the process of refactoring - this was just getting it to work first